src/main/resources/application.properties

```
@@ -9,8 +9,8 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.generate-ddl=true
 
-server.port=8080
-
 spring.datasource.username=postgres
 spring.datasource.password=postgres
 
+springdoc.swagger-ui.path=/swagger-ui
+server.servlet.context-path=/minigames/crosswordpuzzle/api/v1
```

```suggestion
server.servlet.context-path=/api/v1
```
Leave the rest to the nginx or similar proxies, otherwise the frontend and DevOps guys will be angry again...